addons/edi/models/edi_document.py

```diff
@@ -249,10 +267,17 @@ def _compute_input_count(self):
 
     @api.depends("output_ids", "output_ids.res_id")
     def _compute_output_count(self):
-        """Compute number of output attachments (for UI display)"""
+        """Compute number of output attachments (for UI display)."""
         for doc in self:
             doc.output_count = len(doc.output_ids)
 
+    def _compute_progress_log(self):
```

Comment:
Is it thought to have a many2one field on document doc_progress_id and progress_log a related field with store=False.
From the code it looks like there can be more than one edi doc progress for same edi document

Reply:
One that stores everything is fine.

Reply:
I think we are handling on the code to make sure that there is only one edi document progress per edi, so is ok i believe like it is